plugins/offers.c

```diff
@@ -66,6 +66,49 @@ struct gossmap *get_gossmap(struct plugin *plugin)
 	return global_gossmap;
 }
 
+/* BOLT #12:
+ *   - if it is connected only by private channels:
+ *     - MUST include `offer_paths` containing one or more paths to the node
+ *       from publicly reachable nodes.
+ */
```

Comment:
CI checks is failing here

```
cc common/utils.c
ld tools/check-bolt
plugins/offers.c:69:unknown bolt 12
make: *** [Makefile:523: bolt-check/plugins/offers.c] Error 1
rm external/build-x86_64-linux-gnu/libwally-core-build/src/secp256k1/libsecp256k1.la
```

Reply:
It looks like this should be fine once #7887 is merged and the default bold commit is updated to 89b99a578f52fcbabb39417f1855900e985b51db.